ch07/ex7_31.h

```diff
@@ -6,8 +6,11 @@
 //  Copyright (c) 2014 pezy. All rights reserved.
 //
 
-#ifndef CP5_ex7_31_h
+#ifndef CP5_ex7_31_h    
 #define CP5_ex7_31_h
+/*i use #ifndef 7_31_H and there is an error:macro names must be identifiers
```

Comment:
The standard (C11 6.4.2 Identifiers) mandates that identifiers **must not start with a digit**.